ospd/parser.py

```diff
@@ -219,9 +219,27 @@ def __init__(self, description: str) -> None:
                 'Default %(default)s'
             ),
         )
+        parser.add_argument(
+            '--feed-updater',
+            default="openvas",
+            type=str,
+            help=(
+                'Sets the method of updating the feed.'
+                ' Can either be openvas or nasl-cli.'
+                ' Default: %(default)s.'
+            ),
+        )
 
         self.parser = parser
 
+    def feed_updater(self, string: str) -> str:
```

Comment:
The function isn't used. I guess you want to use it as the type of the `--feed-updater` argument but instead it would be better to just use a [choice in this case](https://docs.python.org/3/library/argparse.html#choices).

Reply:
Thanks, your right, I forgot to add it as type but choices are better.